src/components/Todo/Todo.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+export function Todo({
+  completed,
+  title,
+  userId,
+  id,
+  selectedTodoId,
+  handleChange,
+}) {
+  return (
+    <li
+      className={completed
+        ? 'TodoList__item TodoList__item--checked'
+        : 'TodoList__item TodoList__item--unchecked'}
```

Comment:
npm classnames

Reply:
```suggestion
      className={classNames('TodoList__item', {
          'TodoList__item--checked': completed === true,
          'TodoList__item--unchecked': completed === false,
        })}
```
Почему в данном случае стоит использвать npm пакет classnames? Почему тернарный оператор не подходит?

Reply:
Done